apps/console/src/features/applications/components/settings/attribute-management/attribute-selection-oidc.tsx

```diff
@@ -0,0 +1,823 @@
+/**
```

Comment:
Can we extend the existing attribute-selection component? 
https://github.com/wso2/identity-apps/blob/master/apps/console/src/features/applications/components/settings/attribute-management/attribute-selection.tsx

Reply:
Will resolve this in a future PR.